tests/index-test.ts

```diff
@@ -158,4 +158,41 @@ describe('SonarQube Formatter', () => {
       }
     `);
   });
+
+  it('can format output when there startLine is missing', async () => {
+    await project.setConfig({
+      rules: {
+        'no-bare-strings': 'error',
+      },
+    });
+
+    await project.write({
+      app: {
+        templates: {
+          'application.hbs': '<div></div>',
+        },
+      },
+      '.lint-todo': 'add|ember-template-lint|no-bare-strings|1|5|1|5|70c881d4a26984ddce795f6f71817c9cf4480e79|1687824000000|1690416000000|1693008000000|app/templates/application.hbs',
```

Comment:
Will this result in deterministic output? Other TODO tests generally require you to set the current date to a stable value in order to keep the tests stable.

Reply:
Updated!